celery/local.py

```diff
@@ -148,12 +148,6 @@ def __setitem__(self, key, value):
     def __delitem__(self, key):
         del self._get_current_object()[key]
 
-    def __setslice__(self, i, j, seq):
```

Comment:
can you please verify this removals won't create any regression in the code?

Reply:
These were [deprecated back in the Python 2 days](https://docs.python.org/2.7/reference/datamodel.html#additional-methods-for-emulation-of-sequence-types) and were removed in Python 3, which [passes `slice` objects to the `__getitem__`, `__setitem__` and `__delitem__` methods instead](https://docs.python.org/3/reference/datamodel.html?highlight=__getitem__#object.__getitem__).

I can confirm they are not being used at the moment:

```
>>> from celery.local import Proxy
>>> a = list(range(10))
>>> proxy = Proxy(lambda: a)
>>> print(proxy)
[0, 1, 2, 3, 4, 5, 6, 7, 8, 9]
>>> print(proxy[2:])
[2, 3, 4, 5, 6, 7, 8, 9]
>>> proxy[2:] = [0]
>>> print(proxy)
[0, 1, 0]
>>> del proxy[2:]
>>> print(proxy)
[0, 1]
```